content/wasm-functions/quickstart.md

```diff
@@ -37,7 +37,7 @@ Download the `spin` binary using the `install.sh` script hosted on this site:
 
 <!-- @selectiveCpy -->
 
-<pre class="bash spin-install" id="spin-install-quick"><code>$ curl -fsSL https://wasm-functions.fermyon.app/downloads/install.sh | bash
```

Comment:
Can you update the few places in `prod-monitoring.yaml` that rely on this old URL? https://github.com/fermyon/neutrino/blob/0976f623b3829d5ca353f5114bf36d119f8b6af2/.github/workflows/prod-monitoring.yaml#L62. If we don't update this our monitors are going to break.